### ISSUE TITLE
Any way to change the mp3 bitrate?

### ISSUE THREAD

**Issue by JoshWho** [WWBN/AVideo#9524]
I want to make it 64k bitrate so people with low data can listen to it. Currently it makes it 128 bitrate I believe.

**Comment by DanielnetoDotCom** [comment]
Sorry for now I do not have this option

**Comment by JoshWho** [comment]
Just saying the whole goal of the mp3 was for people that have low bandwidth that couldn't watch the videos. Serving a mp3 file as large as the video kind of defeats the purpose. 

I know it is just a ffmpeg command why not change it to 64k mono or 96k mono?

**Comment by DanielnetoDotCom** [comment]
No, because it will affect all the system, to do it I would need to create an option for it, I may do it in the future but not now.
